spec/features/complete_spec.rb

```diff
@@ -246,6 +243,33 @@ def sign_in_to_gds_sso(username, password, otp_token)
     click_button "Sign in"
   end
 
+  def sign_in_to_auth0(using_auth0_passwordless_connection)
+    # Username is the value entered into the Auth0 email input - it might be a google group
+    auth0_email_username = ENV.fetch("AUTH0_EMAIL_USERNAME") { raise "You must set AUTH0_EMAIL_USERNAME to use Auth0" }
+
+    fill_in "Email address", :with => auth0_email_username
+    click_button "Continue"
+
+    if using_auth0_passwordless_connection
+      info "Logging in using Auth0 passwordless connection"
+
+      # Gmail address and password are the values used to access the gmail account via POP3
+      auth0_gmail_address = ENV.fetch("AUTH0_GMAIL_ADDRESS") { raise "You must set AUTH0_GMAIL_ADDRESS to use Auth0" }
+      auth0_gmail_password  = ENV.fetch("AUTH0_GOOGLE_APP_PASSWORD") { raise "You must set AUTH0_GOOGLE_APP_PASSWORD to use Auth0" }
+
+      code = get_auth0_code(auth0_email_username, auth0_gmail_address, auth0_gmail_password)
+      fill_in "Enter the code", :with => code
+      click_button "Continue"
+    else
+      info "Logging in using Auth0 database connection"
+
+      auth0_user_password  = ENV.fetch("AUTH0_USER_PASSWORD") { raise "You must set AUTH0_USER_PASSWORD to use Auth0" }
+
+      fill_in "Password", :with => auth0_user_password
```

Comment:
As an aside and mentioned in the Trello ticket I think it would be prudent to enable MFA, so we might need to revisit this to generate and supply an MFA code but that's separate to this PR 👍